nncf/quantization/algorithms/min_max/torch_backend.py

```diff
@@ -238,3 +228,34 @@ def _create_quantizer_insertion_command(nncf_graph: NNCFGraph,
         quantizer = PTMinMaxAlgoBackend._create_quantizer(quantizer_config,
                                                           scale_shape, statistics)
         return PTInsertionCommand(target_point, quantizer, TransformationPriority.QUANTIZATION_PRIORITY)
+
+    @staticmethod
+    def get_model_type_ignore_scope(model_type: ModelType):
```

Comment:
Why do we use here the `f_names` instead of just names as it was done for `OV` or `ONNX`?

Reply:
Aligned